presto-main/src/test/java/io/prestosql/server/protocol/TestQueryResultRows.java

```diff
@@ -280,6 +289,62 @@ public void shouldHandleNullTimestamps()
                 .containsExactly(newArrayList(null, null));
     }
 
+    @Test
+    public void shouldHandleNullTimestampsInArray()
```

Comment:
Is the fact that we are testing in TIMESTAMP beneficial in any way. If so please also use timestamp in test related to MAP. Otherwise change the test method `shouldHandleNullValuesInArray`